ui/src/app/styles/cards.css

```diff
@@ -7,3 +7,20 @@
 .card * {
   margin-top: 0rem;
 }
+
+.create-card {
```

Comment:
I'd make this `.card.create-card` to make a bit more specific? To ensure it overrides `.card`